src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -92,7 +92,10 @@ def _getLumiList(self, blockName=None, lfns=None, validFileOnly=1):
             item = {}
             item["RunNumber"] = lumisItem['run_num']
             item['LumiSectionNumber'] = lumisItem['lumi_section_num']
+            if lumisItem.get('event_count', None) is not None:
```

Comment:
same here. Maybe it would be better to just propagate `event_count` (EventCount) everywhere. I'm not sure if there is any performance penalty on the SQL transactions though.